### PR TITLE
Support pool ipam

### PR DESCRIPTION
**Pull request by yaocw2020**
- Support pool ipam
- Delete kube-vip chart

issue: https://github.com/harvester/harvester/issues/633

### CODE REVIEW

**Review by guangbochen — COMMENTED** [review]
the kubevip states `Service LoadBalancer address pools per namespace or global`, does it means each namespace can have a range of IP pool config, I saw u have requested the user to config the `ip_pool` in the CCM but in which I reckon it's better to be configured on the harvester side per namespace(per guest k8s cluster since all VMs run in the same namespace)

**Comment by yaocw2020** [comment]
> the kubevip states `Service LoadBalancer address pools per namespace or global`, does it means each namespace can have a range of IP pool config, I saw u have requested the user to config the `ip_pool` in the CCM but in which I reckon it's better to be configured on the harvester side per namespace(per guest k8s cluster since all VMs run in the same namespace)

Yes. The configurations are stored in the configmap. From my perspective, we can set it in UI.

**Review by gitlawr — APPROVED** [review]
LGTM

File: deploy/harvester-load-balancer/Chart.yaml [hunk 1]
```diff
@@ -20,15 +20,9 @@ version: 0.1.0
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
-appVersion: 0.1.0
+appVersion: v0.1
```
Comment (guangbochen) [inline deploy/harvester-load-balancer/Chart.yaml:23]: the previous app version is accurate in regards to the semantic versioning.

**Review by guangbochen — APPROVED** [review]
LGTM

**Pull request merged**